ironfish/src/primitives/target.ts

```diff
@@ -201,7 +201,7 @@ export class Target {
 
     // max(1 - (current_block_timestamp - parent_timestamp) / 10, -99)
     const diffInSeconds = (time.getTime() - previousBlockTimestamp.getTime()) / 1000
-    const sign = BigInt(Math.max(1 - Math.floor(diffInSeconds / 10), -99))
+    const sign = BigInt(Math.max(1 - Math.floor(diffInSeconds / 40), -99))
```

Comment:
This doesn't take in the TARGET_TIME_MS variable, can we parameterize this to use that?